bravado/client.py

```diff
@@ -271,15 +282,16 @@ def response_callback(incoming_response, operation):
     raise_on_unexpected(incoming_response)
 
     try:
-        swagger_return_value = unmarshal_response(incoming_response, operation)
+        incoming_response.swagger_result = unmarshal_response(
```

Comment:
I actually want to keep it this way so a single object acts as a gateway to all contained concerns. Similiar to the server side with the request object in a pyramid in a view:

```python
@view('blah')
def get_pet(request):
    pet_id = request.swagger_data['pet_id']
```
Also, will make it simpler to pass around just the response to callbacks in #178.